samples/cellular/modem_shell/README.rst

```diff
@@ -1518,6 +1518,15 @@ For example:
 
          west build -p -b nrf9160dk_nrf9160_ns -- -DOVERLAY_CONFIG="overlay-modem-trace-flash.conf;overlay-memfault.conf" -DDTC_OVERLAY_FILE=nrf9160dk_ext_flash.overlay
 
+Enabling TFM logging with modem traces
+=======================================
+In order for TFM logging not to interfere with modem traces, enabling the tfm logging on the same UART as the application utilizes is recommended.
+This can be done by enabling the ``tfm-enable-share-uart`` snippet. This can be done by adding the following command to the west build command:
```

Comment:
```suggestion
=======================================

To prevent TF-M logging from interfering with modem traces, enable TF-M logging on the same UART as the application.
Use the following command:
```